README.md

```diff
@@ -80,15 +80,27 @@ cronjobs:
 With this in your current directory, you can now:
 
 ```sh
-# Bring up a local development environment using docker-compose
+# Bring up a local development environment using docker compose
 ktd up
+```
 
+```sh
 # Deploy the project to a Kubernetes namespace
 kubetools deploy my-namespace
 ```
 
 ## Installing
 
+**NOTE**: before upgrading to version 14.0 or above, you _must_ run `ktd destroy` for all existing 
+local Kubetools projects.
+There is also a bug on the latest version of Docker Desktop that prevents the new Buildkit working with 
```

Comment:
For the "bug" aspect of it:
* it looks like it's a bug of `buildx` or `moby`, but not "Docker Desktop" which is a distribution of Docker. 
* it would be good to point at a/the ticket that is still open, or fixed, not one that's closed as "not planned"
* this section should be moved out of this section which is about "installing" the tool. It should be moved to a troubleshooting section.
* please add the latest version number of Docker( or moby or buildx) at the time of the PR, and that you have confirmed (tested) to have the bug.

Reply:
I have added a better explanation of the bug itself, with a link to the relevant issue in `buildx`, plus a more detailed explanation of what happens if the user is affected by the bug and more detail in the workaround.
I've also cleaned up the CHANGELOG to remove information that shouldn't be there:
[7a8b920](https://github.com/EDITD/kubetools/pull/163/commits/7a8b920979214f3b3731fce5706bfd3f56f72628)